Source/WebCore/platform/xdg/MIMETypeRegistryXdg.cpp

```diff
@@ -79,15 +78,13 @@ Vector<String> MIMETypeRegistry::extensionsForMIMEType(const String& mimeType)
     if (mimeType.isEmpty())
         return { };
 
-    WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN // GLib port
     Vector<String> returnValue;
-    char* extensions[MAX_EXTENSION_COUNT];
-    int n = xdg_mime_get_simple_globs(mimeType.utf8().data(), extensions, MAX_EXTENSION_COUNT);
+    std::array <char*, MAX_EXTENSION_COUNT> extensions;
```

Comment:
`std::array<char*, MAX_EXTENSION_COUNT>`

Reply:
Ugh, sorry, the style checker didn't notice this. Is it worth it a follow-up commit?

Reply:
I think so. You can just land it unreviewed.

Reply:
Addressing here https://github.com/WebKit/WebKit/pull/37200